1.9/administration/debugging/quickstart.md

```diff
@@ -0,0 +1,194 @@
+---
+post_title: Quick Start
+feature_maturity: experimental
+menu_order: 3.3
+---
+
+Use this guide to get started with the `dcos task exec` debugging command.
+
+**Prerequisite:**
+
+- A container launched by using the [DC/OS Universal container runtime](/docs/1.9/usage/containerizers/).
+
+# Run an interactive command inside task container
+You can run interactive commands on machines in your cluster by using the `dcos task exec` command. In this example, the `dcos task exec` command is used to copy a simple script from your local machine to the task container on the node. The script is then administered locally by using the `dcos task exec` command.
+
+1.  Create a Marathon app definition and name it `my-app.json` with the following contents:
+
+    ```bash
+    {
+       "id": "/my-app",
+       "cmd": "sleep 100000000",
+       "cpus": 1
+    }
+    ```
+
+1.  Deploy the service on DC/OS:
+
+    ```bash
+    $ dcos marathon app add my-app.json
+    ```
+
+1.  Get the task ID of the app with this CLI command:
+
+    ```bash
+    $ dcos task
+    ```
+    
+    The output should look similar to this: 
+    
+    ```bash
+    NAME        HOST        USER  STATE  ID                                               
+    my-app  10.0.1.106  root    R    <task_id>
+    ```
+
+1.  Write a script called `hello-world.sh` with the following contents:
+
+    ```bash
+    echo "Hello World"
+    ```
+    
+1.  Upload the script to your task container:
+
+    ```bash
+    $ cat hello-world.sh | dcos task exec -i <task_id> bash -c "cat > hello-world.sh"
+    ```
+    
+1.  Give the file executable permissions:
+
+    ```bash
+    $ dcos task exec <task_id> chmod a+x hello-world.sh
+    ```
+1. Run the script inside of the container:
+
+    ```bash
+    $ dcos task exec <task_id> ./hello-world.sh
+    ```
+
+    The output should look similar to this:
+    
+    ```bash
+    Hello World
+    ```
+
+# Launch a long running interactive Bash session
+
+In this example, a long running [job](/docs/1.9/usage/jobs/) is launched by using the `dcos job run` command, and the `dcos task exec` command is used to launch an interactive Bash shell inside the container of that container.
+
+1.  Deploy and run a job with the DC/OS CLI:
+
+    1.  Create the following job definition and save as `⁠⁠⁠⁠test-job.json`. This specifies a sleep job that runs for `10000000` seconds.
+    
+        ```bash
+        {
+            "id": "task-exec-test",
+            "cmd": "sleep 999999999",
+            "cpus": 0.1,
+            "instances": 1
+        }
+        ```
```

Comment:
I have fixed. I inadvertently severely messed up this job example. I have fixed.